tests/basic/pct.rs

```diff
@@ -94,3 +94,154 @@ fn yield_spin_loop_fair() {
 fn yield_spin_loop_unfair() {
     yield_spin_loop(false);
 }
+
+#[test]
+#[should_panic(expected = "called `Option::unwrap()` on a `None`")]
+// Based on Fig 1(a) of the PCT paper.  We model NULL pointer dereference with an Option unwrap
+fn figure1a_pct() {
+    const COUNT: usize = 5usize;
+    // n=2, d=1, so probability of finding the bug is at least 1/2
+    // So probability of hitting the bug in 20 iterations = 1 - (1 - 1/2)^20 > 99.99%
+    let scheduler = PctScheduler::new(1, 20);
+    let runner = Runner::new(scheduler, Default::default());
+    runner.run(|| {
+        let t1 = Arc::new(Mutex::new(None));
+        let t2 = Arc::clone(&t1);
+
+        thread::spawn(move || {
+            for _ in 0..COUNT {
+                thread::yield_now();
```

Comment:
Do these tests still work the way you expect now that `yield_now` deprioritizes?

Reply:
Yeah, I was working on these before you PR.  Will replace.